internal/handler/Handler.go

```diff
@@ -95,7 +95,7 @@ func (h *Handler) getEmployees(w http.ResponseWriter, r *http.Request) {
 	}
 	_, er := w.Write(jsonBytes)
 	if er != nil {
-		http.Error(w, "Error: bad request", http.StatusBadRequest)
+		http.Error(w, "Error: bad request", http.StatusInternalServerError)
```

Comment:
The text of the error also has to be updated

Reply:
Also re check the other similar places

Reply:
+